packages/cli/templates/webcomponents/igc-ts/card/default/files/src/app/__path__/__filePrefix__.ts

```diff
@@ -20,7 +20,7 @@ export default class $(ClassName) extends HTMLElement {
     this.innerHTML = `
```

Comment:
Also, general: So the default Home component is `home.component` but the rest of our templates are simply `card` and so on? That's inconsistent at least and then there's Lint:
![image](https://user-images.githubusercontent.com/3198469/152115177-4cc0c07f-e653-4ff8-ad32-b796779801e7.png)

Not that I'm saying `home.component` passing the check is okay (it's _not_), but first perhaps our components should be consistent and then they shouldn't break the lint rules

Reply:
home.component.ts and not-found.component.ts are renamed to home.ts and not-found.ts, respectively.
I logged issue #997 for the missing .js extensions.